README.md

```diff
@@ -26,6 +26,38 @@ Get started using the documentation below for each respective stack.
 
 ### Hono (Backend)
 
+#### Database Schema
+
+```mermaid
+erDiagram
+    items {
+        Int id
+        String name
+        String brand
+        String photo
+        Int type
+        Int rating
+        String timestamp
+        Int author_username
+    }
+
+    outfits {
+        Int id
+        Int rating
+        String wearDate
+        String author_username
+    }
+
+    items_to_outfits {
+        Int item_id
+        Int outfit_id
+        Int item_type
+    }
+
+    items ||--|{ items_to_outfits : "has outfits"
```

Comment:
items can technically have "zero or more" outfits, not "one or more"

Reply:
Nevermind, was thinking about direct relationship from items to outfits. Relationship holds.